python/ray/serve/tests/test_api.py

```diff
@@ -1012,6 +1012,29 @@ def f(starlette_request):
     }
 
 
+def test_backend_logger(serve_instance):
+    # Tests that that backend logger can be created and used without errors.
+    # Does not test the correctness of the log output.
```

Comment:
It's probably worth testing this to avoid regressions. Can we redirect the logger to a tempfile and read that file or something similar?